src/metiq.py

```diff
@@ -804,7 +804,7 @@ def dump_results(video_results, video_delta_info, audio_results, outfile, debug)
 def z_filter(data, field, z_val):
     mean = data[field].mean()
     std = data[field].std()
-    return data.drop(data[data[field] > mean + z_val * std].index)
+    return data.drop(data[data[field].abs() > abs(mean + z_val * std)].index)
```

Comment:
don't you need to test

    (data[data[field].abs() > abs(mean + z_val * std)) or (data[data[field].abs() < abs(mean - z_val * std))

Reply:
You mean that filtering for unusually small values as well, right?

Reply:
I got it now and I see the problem. How about this instead?
 data[((data[field] - mean)/std).abs() <  z_val]

Reply:
> data[((data[field] - mean)/std).abs() < z_val]

Won't this all the good values? If data[field] is exactly the mean, the abs value is 0, which is likely smaller than any z_val. 

Also, you are using the average of the whole timeseries. We are finding bimodal latency distributions. What we want is to compare the value at time $t$ with the average of the values in the window (t-k, t+k). The worst case is exactly in the value where the mode changes: There the values on the left are all in a different average than the values on the right. But that's OK: We'd just be separating the modes

Reply:
Also, we need to add some tests for this

Reply:
When the value is close to the mean the z val is close to zero. A negative z-value indicates values smaller than the average and a positive z-val indicates larger than average.
Filtering on absolute value should remove outliers in both.

Filtering on with a time window is a different thing. I can add that as a separate functionality.

Reply:
Instead of dropping I just keep the others. It is faster and easier on the eyes.
Changed the names to more reflect what is happening.